airflow/providers/google/marketing_platform/hooks/analytics.py

```diff
@@ -0,0 +1,77 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from typing import Any, Dict, List
+
+from googleapiclient.discovery import Resource, build
+
+from airflow.providers.google.cloud.hooks.base import CloudBaseHook
+
+
+class GoogleAnalyticsHook(CloudBaseHook):
+    """
+    Hook for Google Analytics 360.
+    """
+
+    def __init__(
+        self,
+        api_version: str = "v3",
+        gcp_connection_id: str = "google cloud default",
+        *args,
+        **kwargs
+    ):
+        super().__init__(*args, **kwargs)
+        self.api_version = api_version
+        self.gcp_connection_is = gcp_connection_id
+        self._conn = None
+
+    def get_conn(self) -> Resource:
+        """
+        Retrieves connection to Google Analytics 360.
+        """
+        if not self._conn:
+            http_authorized = self._authorize()
+            self._conn = build(
+                "analytics",
+                self.api_version,
+                http=http_authorized,
+                cache_discovery=False,
+            )
+        return self._conn
+
+    def list_accounts(self) -> List[Dict[str, Any]]:
+        """
+        Lists accounts list from Google Analytics 360.
+
+        More information about API:
+        http://googleapis.github.io/google-api-python-client/docs/dyn/analytics_v3.management.accounts.html
+
+        """
+
+        self.log.info("Retrieving accounts list...")
+
+        # pylint: disable=no-member
+        result = []  # type: List[Dict]
+        conn = self.get_conn()
+        # return self._list_emails_from_response(accounts)
+        while True:
+            request = conn.management().accounts().list(start_index=len(result) + 1)
+            response = request.execute(num_retries=self.num_retries)
+            result.extend(response.get('items', []))
+            if response["totalResults"] >= len(result):
```

Comment:
```suggestion
            if response["totalResults"] == len(result):
```
We want to break the loop once we get all results, right?

Reply:
Yes, below this is a `break`. 
I changed it to `<=` in the latest commit.

Reply:
One more fix no is `==`.